website/markdown/docs/examples/app-clips.mdx

```diff
@@ -0,0 +1,47 @@
+---
+name: App Clips
+excerpt: 'This page documents how to declare and use App Clip targets.'
+---
+
+import Message from '../components/message'
+
+# App Clips
+
+## Product Types
+
+An App Clip is a small part of your app that’s discoverable at the moment it’s needed. App Clips are fast and lightweight so a user can open them quickly.
+
+For example, this is how an AppClip can be declared:
+
+```swift
+let project = Project(name: "App",
+                      targets: [
+                        Target(name: "App",
+                               platform: .iOS,
+                               product: .app,
+                               bundleId: "io.tuist.App",
+                               infoPlist: "App/Configs/Info.plist",
+                               sources: ["App/Sources/**"],
+                               dependencies: [
+                                    .target(name: "AppClip"),
+                                ]),
+                        Target(name: "AppClip",
+                               platform: .iOS,
+                               product: .appClip,
+                               bundleId: "io.tuist.App.Clip",
+                               infoPlist: "AppClip/Configs/Info.plist",
+                               sources: ["AppClip/Sources/**",],
+                               entitlements: "AppClip/Entitlements/AppClip.entitlements",
```

Comment:
What should be the content of the entitlements?

Reply:
the one mentioned in the example doc